### PR TITLE
microsite: add feedback plugin to plugins directory

### PR DESCRIPTION
**Pull request by yashoswalyo**
## Hey, I just made a Pull Request!

We have developed feedback plugin for backstage, which helps to gather feedback for all entities individually. With this PR we are adding it to backstage plugins directory.
#### :heavy_check_mark: Checklist

- [ ] NA. A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] NA. Tests for new functionality and regression tests for bug fixes
- [ ] NA. Screenshots attached (for UI changes) 
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by yashoswalyo** [comment]
Well I saw just now **microsite** added to `.prettierignore` file

File: microsite/data/plugins/feedback.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Feedback
+author: Red Hat
+authorUrl: https://redhat.com
+category: Engage
+description: A plugin for collecting user feedbacks for your application. # Max 170 characters
+documentation: https://github.com/janus-idp/backstage-plugins/tree/main/plugins/feedback#readme
```
Comment (awanlin) [inline microsite/data/plugins/feedback.yaml:7]: Would be great if the screenshots were not hidden, I missed them when I was looking over the documentation. The ones you have a really good and give me a good feel for the functionality right away. This is very helpful for people who are considering using your plugin so making them visible to start would be great!

**Review by awanlin — CHANGES_REQUESTED** [review]
Hi @yashoswalyo, left a pair of comments.

I am curious: why did you build an entirely new plugin instead of contributing these features to the existing [Entity Feedback](https://github.com/backstage/backstage/tree/master/plugins/entity-feedback) plugin?

File: microsite/data/plugins/feedback.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Feedback
+author: Red Hat
+authorUrl: https://redhat.com
+category: Engage
```
[the remark this answers is not in the document]
Comment (yashoswalyo) [reply microsite/data/plugins/feedback.yaml]: Updated the category

File: microsite/data/plugins/feedback.yaml (hunk 1 above)
Comment (yashoswalyo) [reply microsite/data/plugins/feedback.yaml:7]: Cool, I'll update my readme as per your suggestion

**Review by awanlin — APPROVED** [review]

**Comment by awanlin** [comment]
> Entity-feedback as name suggests only takes feedback for catalog entites, We wanted feedback for whole backstage too. Also this new plugin creates jira tickets for bug reports

Thanks for the reply. I had seen the Jira features but thought those would be great in the current Entity Feedback plugin. I totally missed that this could be used more globally, that makes sense but perhaps you need to make that feature more clear? It does make more sense for this to be in it's own plugin given that information.

**Review by freben — APPROVED** [review]

**Pull request merged**